### DISCUSSION TITLE
Make the timezone picker for participants the same as event creation

### DISCUSSION THREAD

**Discussion by kleenkanteen** [lukevella/rallly#780] [General]
When creating an event, you choose the timezone with this component, it is fast and simple:

![image](https://github.com/lukevella/rallly/assets/67878058/36e4b276-346d-4676-8bf7-075578d79b7c)

But when you are a participant and want to choose a timezone, the picker is different. It has a noticeable lag when opening and scorlling. Example event: https://app.rallly.co/invite/Ra7NjCg5Vumf. When you go to a poll and click on the "Times shown" button on the left:

![image](https://github.com/lukevella/rallly/assets/67878058/25f2bc40-6fcd-4210-a5b0-6cdd5bd1c628)

This picker pops up instead:

![image](https://github.com/lukevella/rallly/assets/67878058/fb85b5c9-7e11-4dc7-89dc-9162548bca1b)

It's quite laggy unlike the event creation picker. Ok to change the partipicant timezone picker to the event picker?

**Comment by lukevella** [comment]
I wasn't aware of any performance issues for the time zone picker. The one on the poll creation page is quite old and uses headless-ui components which I plan to remove in favor of radix-ui. Any chance you can take a recording demonstrating the poor performance as I don't see it myself?

**Comment by kleenkanteen** [comment]
Turns out hardware acceleration was off, turning it on made it smooth again. Same result on both chrome and brave.
